readthedocs/gold/static-src/gold/js/checkout.js

```diff
@@ -28,15 +27,27 @@ function StripeCheckoutView(config) {
             });
     };
 
-    self.createCheckoutSession = function (priceId) {
+    self.createCheckoutSession = function () {
+        var priceId = document.getElementById(self.levelId).value;
+        // One-time donation fields
+        var name = document.getElementById('id_name').value;
+        var email = document.getElementById('id_email').value;
+        var logoUrl = document.getElementById('id_logo_url').value;
+        var siteUrl = document.getElementById('id_site_url').value;
+        var public = document.getElementById('id_public').checked;
```

Comment:
These fields will _never_ exist for Gold Membership, but they will for one-time donations. I'm not sure what's the best JS pattern to use here to make them optional.